### ISSUE TITLE
New Accessibility Best Practices Section with Keyboard Accessibility

### ISSUE THREAD

**Issue by gladius-mtl** [SAP/spartacus-docs#662]
Kacper provided input on Keyboard Accessibility and HTML Semantics as part of accessibility best practices. It doesn't have a natural home in any existing document, so let's create a new Accessibility Best Practices Section, and add Kacper's input as a child page to that.
